build/docs.js

```diff
@@ -29,6 +29,32 @@ async function buildDocs() {
       JSON.stringify(docs)
     );
   });
+
+  glob(path.resolve(__dirname, "../src/entityDescriptions/**/*.md"), async (err, files) => {
```

Comment:
Good first step. Let's see if we can refactor this so we're not duplicating code